es/es-server/src/main/java/org/elasticsearch/index/shard/IndexShard.java

```diff
@@ -641,19 +643,21 @@ private IndexShardState changeState(IndexShardState newState, String reason) {
             return new Engine.IndexResult(e, version, opPrimaryTerm, seqNo);
         }
 
-        return index(getEngine(), operation);
+        return index(engine, operation);
     }
 
     public static Engine.Index prepareIndex(DocumentMapperForType docMapper, Version indexCreatedVersion, SourceToParse source, long seqNo,
-            long primaryTerm, long version, VersionType versionType, Engine.Operation.Origin origin, long autoGeneratedIdTimestamp,
-            boolean isRetry) {
+                                            long primaryTerm, long version, VersionType versionType, Engine.Operation.Origin origin,
+                                            long autoGeneratedIdTimestamp, boolean isRetry,
```

Comment:
I think we should adopt our formatting style (either all args on 1 line, or 1 arg per line) when we take over code. 

(Fine for me to ignore this for this PR, but for the future)